app/routes/admin/sessions/list.js

```diff
@@ -0,0 +1,108 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    switch (this.get('params.sessions_state')) {
+      case 'pending':
+        return this.i18n.t('Pending');
```

Comment:
@harshitagupta30 no. Don't directly put it inside the `this.i18n.t` ... We need to distinctly define the strings for translation. Switch case is the way to go here even though it looks very in-efficient.